docs/contributor/04-10-testing-strategy.md

```diff
@@ -0,0 +1,13 @@
+# Testing Strategy
+
+Each pull request to the repository triggers CI/CD jobs that verify the Serverless Operator reconciliation logic and run integration tests of the Serverless module.
+
+- `pre-serverless-operator-operator-build` - Compiling the Serverless Operator code and pushing its docker image.
+- `pre-serverless-operator-operator-tests` - Testing the Serverless Operator reconciliation code (Serverless CR CRUD operations).
+- `pre-main-serverless-operator-verify` - Integration testing for the Serverless module installed by Serverless Operator (not using Lifecycle Manager).
+- `pull-serverless-module-build` - Bundling a module template manifest that allows testing it against Lifecycle Manager manually. 
```

Comment:
```suggestion
- `pull-serverless-module-build` - Bundles a ModuleTemplate manifest that allows testing it against Lifecycle Manager manually. 
```

Reply:
If that's the same [ModuleTemplate](https://github.com/kyma-project/lifecycle-manager/blob/main/README.md#basic-concepts) that we mention in Lifecycle Manager docs, I think we should use upper camel case